app/controllers/claims_controller.rb

```diff
@@ -121,6 +121,8 @@ def handle_magic_link
       journey_session.answers.assign_attributes(email_address: params[:email])
       journey_session.answers.assign_attributes(email_verified: true)
       journey_session.save!
+    else
```

Comment:
So the expired-link page will be shown if the OTP code is not expired, but just not valid - not sure if this is too much of an edge case, as it would require someone to construct their own magic link. It seems an unlikely scenario, and showing the expired-link page in this case seems fine.